tests/main/views/test_buyers.py

```diff
@@ -647,9 +653,10 @@ def test_wrong_lot_get_view_brief_responses(self):
     def test_wrong_lot_publish_brief(self):
         self._load_page("/digital-outcomes/1234/publish", 404)
 
-    def test_wrong_lot_post_delete_a_brief(self):
-        data = {"delete_confirmed": True}
-        self._load_page("/digital-outcomes/1234/delete", 404, method='post', data=data)
+    @pytest.mark.parametrize('action', ['delete', 'withdraw'])
+    def test_wrong_lot_post_delete_or_withdraw_a_brief(self, action):
+        data = {"{}_confirmed".format(action): True}
+        self._load_page("/digital-outcomes/1234/{}".format(action), 404, method='post', data=data)
```

Comment:
This method reads badly in diffs. Doesn't describe what it's doing at all. It's possible to _guess_ that it's asserting it returns 404, but eh. Not really your fault of course... but it'd be nice to make this more transparent at some point.

And huh, having looked at the method, it does _a lot_ for it just being described as 'load page'.

Reply:
I am more than happy to refactor this!

Reply:
On closer inspection, this test class seems mostly redundant now. `_load_page` looks complex, but all that's really happening is a lot of setup (framework/content loader etc)  and a status code assertion for different combinations of lots and frameworks. The individual view classes check these pretty thoroughly. I'll tidy it up as best I can with the aim of removing the class entirely.

Reply:
Done. The diff isn't too bad, in the end. Hopefully the commit message is descriptive enough.